src/gui/sdlmain.cpp

```diff
@@ -2876,6 +2880,10 @@ void Config_Add_SDL() {
 	Pint = Pmulti->GetSection()->Add_int("ysens",Property::Changeable::Always,100);
 	Pint->SetMinMax(-1000,1000);
 
+	Pbool = sdl_sec->Add_bool("rawmouseinput",
+	                          Property::Changeable::OnlyAtStart, false);
```

Comment:
@NicknineTheEagle ; code is looking tight!

@dreamer  - regarding `Pbool` and `Property::Changeable::OnlyAtStart`; I don't think this function's started the migration to cleaned up cases (ie: `pbool`) or the short-hand property timeframes (ie: `constexpr auto on_start = Property::Changeable::OnlyAtStart;`)

Reply:
Maybe? It's best to do them all at once.

Reply:
I agree @NicknineTheEagle.  I know it was one of @dreamer prior comments; will see what he thinks.